src/main/fc/config.c

```diff
@@ -1094,11 +1098,13 @@ void validateAndFixGyroConfig(void)
             motorUpdateRestriction = 0.00003125f;
     }
 
-    if(pidLooptime < motorUpdateRestriction)
-        pidConfigMutable()->pid_process_denom = motorUpdateRestriction / (samplingTime * gyroConfig()->gyro_sync_denom);
+    if (pidLooptime < motorUpdateRestriction) {
+        const uint8_t maxPidProcessDenom = constrain(motorUpdateRestriction / (samplingTime * gyroConfig()->gyro_sync_denom), 1, MAX_PID_PROCESS_DENOM);
+        pidConfigMutable()->pid_process_denom = MIN(pidConfigMutable()->pid_process_denom, maxPidProcessDenom);
```

Comment:
I don't think exceeding `MAX_PID_PROCESS_DENOM` is possible either way, since none of the two values can exceed it.

But if we want to constrain `pid_process_denom` to the maximum possible loop speed it should be MAX. (denom is inverse to speed).